prime-router/settings/prod/0030-hhsprotect.yml

```diff
@@ -0,0 +1,34 @@
+# Run ./prime receiver set --input ./settings/prod/0030-hhsprotect.yml --env prod
+# The change is additions to the juris filter.
+# Note that the processing_mode_code filter is still set to allow "T" data, for now, in Prod.
+---
+name: "elr"
+organizationName: "hhsprotect"
+topic: "covid-19"
+translation: !<CUSTOM>
+  schemaName: "hhsprotect/hhsprotect-covid-19"
+  format: "CSV"
+  defaults: {}
+  nameFormat: "STANDARD"
+  receivingOrganization: null
+  type: "CUSTOM"
+jurisdictionalFilter:
+- "matches(sender_id, SafeHealth,ImageMover,Cue,InBios,Strac)"
+qualityFilter:
```

Comment:
Change to allow Strac data to flow.   These names match the default names set in the respective .schema files for each sender.